content/en/account_management/guide/hourly-usage-migration.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Migrating from the V1 Hourly Usage APIs to V2
+kind: guide
+further_reading:
+- link: "/account_management/plan_and_usage/"
+  tag: "Documentation"
+  text: "Plan and Usage Settings"
+---
+
+## Summary
+Users of the v1 APIs should recognize familiar concepts in the v2 hourly usage API,
+albeit represented in slightly different format.
+
+The most notable differences between the v1 API and the v2 API are that the v2 API:
+* Consolidates all products to one endpoint
+* Follows the JSON:API standard
+* Is paginated
+* Can return data for multiple organizations per request
```

Comment:
@RolfBates @buraizu can we update this to include multi-region support? Maybe say: 
"Can return data for multiple organizations and regions per request"

Reply:
Sure. I can update this and the other section below.